docs/content/node-operation/monitoring-nodes.mdx

```diff
@@ -3,19 +3,19 @@ title: Monitoring Node Health
 sidebar_title: Node Monitoring
 ---
 
-<Callout type="info" title="Logs">
```

Comment:
Callout do not work on the new doc site

Reply:
It seems like the text still appears, just without any extra styling. So no need to rush and update other `Callout`s.